setup.py

```diff
@@ -29,14 +30,19 @@
     name='geoips',
     version=version,
     packages=setuptools.find_packages(),
+    include_package_data=True,
+    package_data={"geoips": ["yaml_configs/*", "yaml_configs/*/*", "yaml_configs/*/*/*",
+                             "image_utils/ascii_palettes/*"]},
     install_requires=['pyresample',           # Base requirement - efficiency improvements >= 1.22.3
                       'numpy',                # Base requirement
                       'xarray',               # Base requirement
                       'matplotlib',           # Base requirement
                       'scipy',                # Base requirement
                       'netcdf4',              # Base requirement
                       'pyyaml',               # Base requirement
-                      # 'cartopy==0.20.2',    # Currently must install via conda
+                      'pyshp>=2.2.0',
+                      'shapely@https://github.com/shapely/shapely/releases/download/1.8.2/Shapely-1.8.2.tar.gz',
+                      'cartopy==0.20.3',    # Currently must install via conda
```

Comment:
This is exciting!  Can take out "currently must install via conda" comment now?

Reply:
Yeah, I think so. Though you may still want to do Cartopy with Conda since the lat/lon label issue isn't fixed yet.